src/libs/ajax/leonardo/Cookies.ts

```diff
@@ -0,0 +1,35 @@
+import { withAuthSession } from 'src/auth/auth-options';
+import { fetchLeoOnce } from 'src/libs/ajax/fetch/fetchLeoOnce';
+
+import { LeoCookiesDataClient, makeLeoCookiesDataClient } from './LeoCookiesDataClient';
+
+const makeCookiesHelper = (deps: CookiesHelperDeps) => (signal?: AbortSignal) => {
+  const { api } = deps;
+
+  const cookiesHelper = {
+    // setAzureCookie: async (proxyUrl: string): Promise<void> => {
+    //   await api.setAzureCookie(proxyUrl);
+    // },
+    // setCookie: async (): Promise<void> => {
+    //   await api.setCookie();
+    // },
+    unsetCookie: async (): Promise<void> => {
+      await api.unsetCookie({ signal });
+    },
+  };
+
+  return cookiesHelper;
+};
+
+type CookiesHelperDeps = {
+  api: LeoCookiesDataClient;
+};
+
+export const Cookies = makeCookiesHelper({
```

Comment:
I'm thinking LeoCookies would be better name for this export and the filename of this module too.